src/index.html

```diff
@@ -17,6 +31,88 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="menu">
+        <a
+          class="header-link"
+          href="#"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="header_icon"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="navbar">
+            <li class="nav-item">
+              <a
+                class="nav-link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav-item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav-item">
```